modules/kubientBidAdapter.md

```diff
@@ -6,21 +6,56 @@
 ​
 # Description
 ​
-Connects to Kubient KSSP demand source to fetch bids.  
+Connects to Kubient KSSP demand source to fetch bids. Banners and Video supported
 ​
-# Test Parameters
+# Banner Test Parameters
+```
+var adUnits = [
+    {
+        code: 'banner-ad-unit',
+        mediaTypes: {
+            banner: {
+                sizes: [[300, 100]]
+            }
+        },
+        bids: [{
+            bidder: 'kubient',
+            params: {
+                zoneid: "5fbb948f1e22b",
+            }
+        }]
+    }
+];
+​
+# Video Test Parameters
+​
+var adUnits = [
+    {
+        code: 'video-ad-unit',
+        mediaTypes: {
+            video: {
+                playerSize: [300, 250],               // required
+                context: 'instream',                  // required
+                mimes: ['video/mp4','video/x-flv'],   // required
+                protocols: [ 2, 3 ],                  // required, set at least 1 value in array
```

Comment:
PR looks good! Last thing to update would be to add a check for these required video properties in isBidRequestValid.

Reply:
@alexartwww did you saw the above question? please add check for required prameters